plugins/modules/azure_rm_storageaccount.py

```diff
@@ -1176,7 +1241,6 @@ def account_obj_to_dict(self, account_obj, blob_mgmt_props=None, blob_client_pro
             default_to_o_auth_authentication=account_obj.default_to_o_auth_authentication,
             allow_cross_tenant_replication=account_obj.allow_cross_tenant_replication,
             allow_shared_key_access=account_obj.allow_shared_key_access,
-            network_acls=account_obj.network_rule_set,
```

Comment:
It seems the model of the API matches this plugin, why do we need a special handling at L1341?